docs/chrome-extension/using-the-chrome-extension.md

```diff
@@ -37,3 +84,23 @@ _note: This feature is not available on private repositories._
 ### Quick Access to important OpenSauced links
 
 ![Quick Access to OpenSauced](../../static/img/extension-links.png)
+
+### Posting highlight
+
+You can access the post to highlight from the main popup in the extension, and when you click on it, it will automatically, get the the PR or issue  title, here you can use our AI functions to get a summary of the highlight similar to the PR summary. 
```

Comment:
```suggestion
You can access the post to highlight it from the popup window in the extension. When you clicked, it will automatically, populate the PR or issue title, here you can use our AI functions to get a summary of the highlight similar to the PR summary. 
```